02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,39 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css">
+  <title>Khadijah's Bio</title>
 </head>
 <body>
-  <h1>Indian Restaurant</h1>
+  <h1><u>Khadijah's Bio</u></h1>
+  <a ref="">I own a company called Best Lead Generation that
+    specializes in providing <div> business-to-business lead generation
+    services to companies in order to help <div>grow their business.</a>
```

Comment:
On lines 9 and 10 you have wrapped some of your paragraph text in a `<div>` tag but I don't see why. If you are going to add styling to some text, it would be best to wrap the text in a `<span>` tag.